vas2nets/tasks.py

```diff
@@ -34,3 +35,26 @@ def run(self, date, **kwargs):
 
 
 fetch_voice_data = FetchVoiceData()
+
+
+class FetchVoiceDataHistory(Task):
+
+    def run(self, start_date, end_date, **kwargs):
+
+        start_date = datetime.strptime(start_date, '%Y-%m-%d')
+        end_date = datetime.strptime(end_date, '%Y-%m-%d') + timedelta(1)
+
+        def daterange(start, end):
+            for n in range(int((end - start).days)):
+                yield start + timedelta(n)
+
+        for day in daterange(start_date, end_date):
+            if day.weekday() < 5:
```

Comment:
I'm not entirely sure why we're only fetching weekdays? Especially since Friday is a big send day, and it's possible that some sends might've gone past midnight in the past?